netbox/templates/core/plugin.html

```diff
@@ -0,0 +1,97 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load form_helpers %}
+{% load i18n %}
+
+{% block title %}{{ plugin.name }}{% endblock %}
+
+{% block object_identifier %}
+{% endblock object_identifier %}
+
+{% block breadcrumbs %}
+  <li class="breadcrumb-item"><a href="{% url 'core:plugin_list' %}">{% trans "Plugins" %}</a></li>
+{% endblock breadcrumbs %}
+
+{% block subtitle %}
+  <div class="text-secondary fs-5">
+    {{ plugin.tag_line }}
+    <a href="{{ plugin.homepage.url }}" target="_blank">Learn more <i class="mdi mdi-launch"></i></a><br />
+    <strong>License:</strong> {{ plugin.license_type }}
+  </div>
+{% endblock subtitle %}
+
+{% block controls %}
+<div class="text-secondary">
+From
+</div>
+{{ plugin.author }}
+{% endblock %}
+
+{% block tabs %}
+  <ul class="nav nav-tabs" role="tablist">
+    <li class="nav-item" role="presentation">
+      <a class="nav-link active" id="overview-tab" data-bs-toggle="tab" data-bs-target="#overview" type="button" role="tab" aria-controls="edit-form" aria-selected="true">
+        {% trans "Overview" %}
+      </a>
+    </li>
+    {% if not plugin.is_local %}
+      <li class="nav-item" role="presentation">
+        <button class="nav-link" id="version-history-tab" data-bs-toggle="tab" data-bs-target="#version-history" type="button" role="tab" aria-controls="object-list" aria-selected="false">
+          {% trans "Version history" %}
+        </button>
+      </li>
+      {% if plugin.is_community %}
```

Comment:
It's not the plugin itself we need to check here, but rather the edition of NetBox. Only the Community edition should include instructions for local installation; Cloud & Enterprise will have different means of installing the plugin.